admin-dev/themes/new-theme/js/components/multiple-choice-table.ts

```diff
@@ -22,6 +22,7 @@
  * @copyright Since 2007 PrestaShop SA and Contributors
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  */
+import ComponentsMap from './components-map';
```

Comment:
```suggestion
import ComponentsMap from '@components/components-map';
```
I think it's preferable to always use our namespaces (since we have them) rather than relative paths It makes the code easier to be copy pasted in external projects